arch/arm/core/aarch32/cpu_idle.S

```diff
@@ -62,19 +62,36 @@ SECTION_FUNC(TEXT, arch_cpu_idle)
 #endif /* CONFIG_ARMV6_M_ARMV8_M_BASELINE */
 #endif /* CONFIG_TRACING */
 
```

Comment:
nit: can you fix indentation on line 59?

Reply:
Not related to this patch, but will add a commit here to fix that.

Reply:
Added a commit for overall whitespace clean-up.

Reply:
thanks